test/runner.py

```diff
@@ -2,8 +2,6 @@
 import os
 import sys
 
-sys.path.insert(0, os.path.dirname(os.path.dirname(__file__)))
```

Comment:
no, I mean, if you have installed an old version of vsc-base but developing on a new version, this will make sure you're not accedentaly testing the installed version. But I guess that's exactly what you don't want to happen on jenkins?